### PR TITLE
Introducing Ganache 7 blog post

### PR DESCRIPTION
**Pull request by kevinbluer**

### CODE REVIEW

File: src/blog/data.json [hunk 1]
```diff
@@ -1,4 +1,11 @@
 {
+  "introducing-ganache-7": {
+    "title": "Introducing Ganache 7",
```
Comment (davidmurdoch) [inline src/blog/data.json:3]: Looks like we still need the "circle" image

**Review by davidmurdoch — CHANGES_REQUESTED** [review]

File: src/blog/data.json (hunk 1 above)
Comment (kevinbluer) [reply src/blog/data.json:3]: Cc @CombsieZ

Comment (davidmurdoch) [reply src/blog/data.json:3]: I've added a temporary image so we can ship

**Review by davidmurdoch — APPROVED** [review]

**Review by cds-amal — APPROVED** [review]
LGTM!

**Pull request merged**